build.gradle

```diff
@@ -55,7 +55,7 @@ allprojects {
       }
     }
     // uncomment if snapshots access is needed
-    /*maven {
+    maven {
```

Comment:
Should we make it conditional?

Reply:
Yes, this should definitely be conditional and only set when `mapbox_navigation_native_upstream` flag is set.